server/controllers/budgetController.js

```diff
@@ -1,74 +1,71 @@
-const Pool = require("../database/connection");
+const budget = require("../models/budgetModel");
+const Transaction = require("../models/transactionModel");
+const Category = require("../models/categoryModel");
 
 module.exports = {
+  getBudget: async (req, res) => {
```

Comment:
assign to `const BudgetContoller `and export the module ,  then no need to import as seperate methods

Reply:
fixed in commit 099eb1dd4c684658b8f51ed33da07bae864438a8